app/portainer/components/multi-endpoint-selector/multi-endpoint-selector.js

```diff
@@ -6,5 +6,6 @@ angular.module('portainer.app').component('multiEndpointSelector', {
     endpoints: '<',
     groups: '<',
     tags: '<',
+    onSearch: '<',
```

Comment:
changes to this component are not needed as we ended up using group-association-table